index.html

```diff
@@ -232,5 +257,43 @@ <h2 class="box-h2">
             <img class="box-img" src="./assets/skill.png" alt="skills">
         </div>
     </section>
+    <!-- Contact Me-->
+    <section class="section-bg">
+        <h1>
+            Get Started
+        </h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+            dolore magna.
+        </p>
+        <form action="https://formspree.io/f/xoqbpaza" method="POST" action="submit">
```

Comment:
- [ ] [**_Required_**] as mentioned by the previous reviewer, the form content should use display flex, I would suggest adding these CSS styling : 

```CSS
form {
    display: flex;
    flex-direction: column;
}
```

- [ ] [OPTIONAL] since the previous reviewer forgot to mention this, but the contact me section should use flex as well, I highly recommend implementing that as well by adding : 

```CSS
.section-bg {
    padding-top: 68px;
    padding-bottom: 68px;
    padding-left: 16px;
    padding-right: 16px;
    background-color: #f9f9fa;
    background-image: url(./assets/geo.png);
    background-repeat: no-repeat;
   /* Adding Flexbox */
    display: flex;
    flex-direction: column;
}
```